src/pubsub/mod_pubsub.erl

```diff
@@ -240,6 +245,78 @@ stop(Host) ->
     gen_server:call(Proc, stop),
     ejabberd_sup:stop_child(Proc).
 
+-spec config_spec() -> mongoose_config_spec:config_section().
+config_spec() ->
+    #section{
+       items = #{<<"iqdisc">> => mongoose_config_spec:iqdisc(),
+                 <<"host">> => #option{type = string,
+                                       validate = domain_template},
+                 <<"backend">> => #option{type = atom,
+                                          validate = {module, mod_pubsub_db}},
+                 <<"access_createnode">> => #option{type = atom,
+                                                    validate = access_rule},
+                 <<"max_items_node">> => #option{type = integer,
+                                                 validate = non_negative},
+                 <<"max_subscriptions_node">> => #option{type = integer,
+                                                         validate = non_negative},
+                 <<"nodetree">> => #option{type = binary,
+                                           validate = {module, nodetree}},
+                 <<"ignore_pep_from_offline">> => #option{type = boolean},
+                 <<"last_item_cache">> => #option{type = atom,
+                                                  validate = {enum, [mnesia, rdbms, false]}},
+                 <<"plugins">> => #list{items = #option{type = binary,
+                                                        validate = {module, node}}},
+                 <<"pep_mapping">> => #list{items = pep_mapping_config_spec()},
+                 <<"default_node_config">> => default_node_config_spec(),
+                 <<"item_publisher">> => #option{type = boolean},
+                 <<"sync_broadcast">> => #option{type = boolean}
+                }
+      }.
+
+pep_mapping_config_spec() ->
+    #section{
+       items = #{<<"namespace">> => #option{type = binary,
+                                            validate = non_empty},
+                 <<"node">> => #option{type = binary,
+                                       validate = non_empty}},
+       required = all,
+       process = fun ?MODULE:process_pep_mapping/1
+      }.
+
+default_node_config_spec() ->
```

Comment:
I've added a backlog documentation item to add a list of node options that can be configured.

Reply:
Good, thanks